cinema/views.py

```diff
@@ -97,12 +100,25 @@ def get_queryset(self):
     def get_serializer_class(self):
         if self.action == "list":
             return MovieListSerializer
-
         if self.action == "retrieve":
             return MovieDetailSerializer
-
+        if self.action == "upload_image":
+            return MovieImageSerializer
```

Comment:
The `upload_image` action correctly uses the `MovieImageSerializer` to handle image uploads. Ensure that this is the only method to upload images, as per the task requirements.